tests/test_aktualizr_kill.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import logging
+import argparse
+
+from os import getcwd, chdir
+
+from test_fixtures import with_aktualizr, with_uptane_backend, KeyStore, with_secondary, with_path,\
+    DownloadInterruptionHandler, MalformedJsonHandler, with_director, with_imagerepo, InstallManager,\
+    with_install_manager, with_images, MalformedImageHandler, with_customrepo, SlowRetrievalHandler, \
+    RedirectHandler, with_sysroot, with_treehub
+
+
+logger = logging.getLogger(__file__)
+
+
+@with_uptane_backend(start_generic_server=True, port=8891)
```

Comment:
This commit should be synced with this one https://github.com/advancedtelematic/aktualizr/pull/1395 as I changed in it an approach to choosing a port for the mocked servers.